reports/aksversions/tests/test_dummy.py

```diff
@@ -0,0 +1,2 @@
+def test_dummy():
+    assert True
```

Comment:
possible to put a real test so that there can be confidence when merging renovate PRs?